common/src/main/java/com/wynntils/models/lootrun/scoreboard/LootrunScoreboardPart.java

```diff
@@ -27,6 +27,8 @@ public class LootrunScoreboardPart extends ScoreboardPart {
     private static final Pattern DESTROY_PATTERN = Pattern.compile("^Destroy the objective!$");
     private static final Pattern DEFEND_PATTERN = Pattern.compile("^Defend for (\\d+)s!$");
 
+    private static final Pattern TIMER_PATTERN = Pattern.compile("^[-—] Time Left: (\\d+):(\\d+)$");
```

Comment:
Ok, this can be "§f(\d{2}):\d{2}"

Reply:
@Rafii2198 This should work now, sorry it took a while my regex isn't great (:

Reply:
I tried doing it, and it wouldn't work for some reason... but well it works